misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/util/TesterinaUtils.java

```diff
@@ -198,10 +196,7 @@ public static void setManifestConfigs(Path sourceRoot) {
      */
     public static void executeTests(Path sourceRootPath, Map<BLangPackage, CompiledBinaryFile.ProgramFile>
             programFileMap) {
-        // Load configuration file. The default config file is taken "ballerina.conf" in the source root path
-        LauncherUtils.loadConfigurations(sourceRootPath, new HashMap<>(), null, false);
```

Comment:
Why was this removed? Wouldn't this cause problems if the user runs `ballerina test` with a config file provided?